#4*3*8+4*2 = 104bytes 
float32[8] em_pos_fb_raw
float32[8] em_vel_fb_raw
float32[8] em_trq_fb_raw
uint32[2]  em_state